dashboard/lib/tasks/seed.rake

```diff
@@ -148,15 +147,16 @@ namespace :seed do
     update_scripts(script_files: script_files)
   end
 
-  task scripts: SCRIPTS_DEPENDENCIES do
+  task scripts: :environment do
+    time_subtasks(SCRIPTS_DEPENDENCIES)
     update_scripts(incremental: false)
   end
 
-  task scripts_incremental: SCRIPTS_DEPENDENCIES do
+  task scripts_incremental: SCRIPTS_DEPENDENCIES + [:environment] do
```

Comment:
I may be misreading, but it doesn't look necessary to explicitly include `[:environment]` here and below. worth trying without?

Reply:
I added it because it was previously included in `SCRIPT_DEPENDENCIES` - does it look to you like it shouldn't be needed? If so I can try without.

Reply:
I mentioned it because it looks like many subtasks already depend on it. more generally, why is it important to exclude from the timing measurements?